phidgets_high_speed_encoder/msg/encoder_params.msg

```diff
@@ -0,0 +1,5 @@
+Header header
+int32 index
+int32 count
+int32 count_change
+int32 time
```

Comment:
To conform with ROS naming standards, this message should be called EncoderParams.